src/index.html

```diff
@@ -3,12 +3,57 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family= Roboto: wght@400;500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body id="home">
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          src="\images\logo.png"
+          alt="logo"
+          class="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="#APPLE" class="link_menu is-active">APPLE</a>
+          </li>
+          <li class="nav_item">
+            <a href="#SAMSUNG" class="link_menu">SAMSUNG</a>
+          </li>
+          <li class="nav_item">
+            <a href="#SMARTPHONES" class="link_menu">SMARTPHONES</a>
```

Comment:
```suggestion
            <a href="#SMARTPHONES" class="link_menu">
              SMARTPHONES
            </a>
```
you may split long lines